tasks/orca_catalog_reporting_dummy/test/unit_tests/test_orca_catalog_reporting_dummy_unit.py

```diff
@@ -26,6 +26,8 @@ def test_task_output_json_schema(self):
         """
         result = orca_catalog_reporting_dummy.handler(
             {
+                "pageIndex": 0,
+                "endTimestamp": "2021-10-08T19:24:07.605323Z"
```

Comment:
if we are changing the pattern in my comment above, this timestamp should also be changed